### PR TITLE
GstGtk 1.6.0 (new formula)

### PR DESCRIPTION
**Pull request by LecrisUT**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

An alternative for getting `gtk+3` support to gstreamer.
It's similar to the `gstreamer1.0-gtk` patch on the other linux packages, however it is a MUCH cruder approach. Any alternatives are welcome.

Basically this is `gst-plugins-good` building nothing but `gtk`, by manually disabling everything else.

### CODE REVIEW

**Comment by fxcoudert** [comment]
@LecrisUT what are you trying to achieve? How is that used by other formulas/software?

**Comment by LecrisUT** [comment]
Simply to add `gtk+3` support to `homebrew gstreamer`. I have already suggest a more simple and direct change to `gst-plugins-good` and I'm adding this one so the devs can choose which implementation is better.

As for the application, `pdfpc` is built specifically on `gtk+3` so in order to add movie support to the pdf presentation, this support has to be added.
